Controllers/Backend/MollieOrders.php

```diff
@@ -189,6 +195,11 @@ public function partialRefundAction()
             if (empty($order))
                 $this->returnError('Order not found');
 
+            # switch to the config and api key of the shop from the order
```

Comment:
the following code exists three times... probably extracting it to a prif would reduce redundancy?

```
 # switch to the config and api key of the shop from the order
            $shopSwitcher = new MollieShopSwitcher($this->container);
            $this->config = $shopSwitcher->getConfig($order->getShop()->getId());
            $this->apiClient = $shopSwitcher->getMollieApi($order->getShop()->getId());
```

Reply:
this is the integration point and is also assigend different variables.
so the shop switcher is already the extracted code
also this is a small solution to avoid side effects. a better centralized access for api client and config will be done in upcoming refactorings, so for now i would leave it like this

Reply:
as it is already planned to refactor this, I will solve the issue as accepted